.gitignore

```diff
@@ -1,6 +1,7 @@
+.cache
 .sconsign.dblite
-config.log
 .sconf_temp
+config.log
```

Comment:
Is there a reason to change where ```config.log``` appears in the file?

Reply:
I prefer having the .scons* files together